primefaces/src/main/resources/META-INF/resources/primefaces/locales/locale-es.js

```diff
@@ -116,26 +116,26 @@ if (window.PrimeFaces) {
     day: "Día",
     list: "Agenda",
     allDayText: "Todo el día",
-    moreLinkText: "Más....",
+    moreLinkText: "Más...",
     noEventsText: "No hay eventos",
     aria: {
-      "datatable.sort.ASC": "ordenar columna ascendente",
-      "datatable.sort.DESC": "ordenar columna descendente",
-      "datatable.sort.NONE": "eliminar la clasificación en la columna",
+      "datatable.sort.ASC": "ordenar columna ascendentemente",
+      "datatable.sort.DESC": "ordenar columna descendentemente",
+      "datatable.sort.NONE": "eliminar la ordenación de la columna",
```

Comment:
"Eliminar el orden de la columna", IMO.

Reply:
In that context, 

'Eliminar el orden de la columna' and 'eliminar la ordenación de la columna' They have same meaning. More polite/formal the second.

@SoyDiego you need to maintain the same context in the text. You use formal text or expression in some texts, but not in others.

Reply:
The translation in my opinion should not be VERY polite and for example in others Spanish (Argentinian) translations, ''ordenación'' is wrong. Search another website or translations in other websites and you will see your text are super super polite and are not like the typical translations.

Reply:
'Argentinian', thas the problem. You make an es_AR traslation, noa an es_ES traslation.

Reply:
> 'Argentinian', thas the problem. You make an es_AR traslation, not an es_ES traslation.
> 
> In es_ES, the formal/polite text is 'eliminar la ordenación de la columna'.

Is not a problem. Always the translations we're ok in all the websites. Is the first time that I read some translations like yours.  And yes I'm Argentinian but I'm living in Spain since 2011 and I know when translations are not ok. Should be more neutral.